content/presentation-formats/keynote.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Keynote
+---
+
+Keynote templates are available for download to the members of the GitHub organization on [Google Drive](https://drive.google.com/open?id=1Wp3NyCYM-FsU-4MKSbPcBgIsWVgvUQy0).
+
+![The Keynote files](https://user-images.githubusercontent.com/10384315/56326001-35762780-6129-11e9-915d-949763cb1186.png)
+
+### Creating a new presentation
+
+A new presentation can be created by opening `github-presentation-template.key` or `github-presentation-theme.kth`.
+
+**Note:** If you just open the regular keynote file (`.key), you will need to duplicate this for your own presentation if you do not want to override the template for later use. It is recommended that for your first presentation you open the theme file (`.kth`) and add the theme to your Keynote to use for future use.
+
+### Installing the Keynote theme
+
+To install, open the theme file (`.kth`) in Keynote and click **Add to Theme Chooser**.
```

Comment:
I noticed that here you made the button name bold (which follows the GH styleguide) but in other places you made tab and button names in code (in think in Images), may be nice to make it consistent.

Reply:
Good catch! I've made everything use **bold** text.